css/main.css

```diff
@@ -1,3 +1,201 @@
 /* Add your custom CSS here */
+@import url('https://fonts.googleapis.com/css?family=PT+Sans&display=swap');
+body {
+    font-family: 'PT Sans', sans-serif;
+    background: rgb(219, 218, 218);
+    color: sienna;
+}
+/* custom gridfor about-us */
+.row-2 {
+    display: flex;
+    flex-wrap: wrap;
+    margin-right: 0px;
+    margin-left: 0px;
+  }
 
-/* Dont' forget to link this file to your HTML in the <head> */
+  .about-col-6 {
+    width: 100%;
+    position: relative;
+    min-height: 1px;
+    padding-right: 0px;
+    padding-left: 0px;
+  }
+
+  @media (min-width: 768px) {
+    .about-col-6 {
+      width: 50%;
+    }
+}
+
+
+
+h1{
+    color: white;
+    font-size: 4rem;
+}
+
+h2{
+    
+    text-align: center;
+    color: rgb(129, 48, 11);
+    padding-top: 10px;
+}
+
+h3{
+    color: white;
+    font-weight: lighter;
+}
+
+h4{
+    text-align: center;
+    padding: 10px;
+}
+h5{
+    padding: 5px 10px ;
+}
+P{
+    text-align: center;
+    color: rgb(247, 243, 244);
+}
+
+.form-title{
+    color: white;
+    font-size: 2rem;;
+}
+
+img{
+    width: 100%;
+}
+
+.top-nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+}
+
+.content{
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+
+}
+ul{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    padding: 5px;
+}
+
+li {
+    display: inline-block;
+    margin-left: 1.563rem;
+}
+a:hover{
+    color: black;
+}
+
+.hero{
+    
+    height: 50vh;
```

Comment:
By defining the height, when the screen gets to very small, the button on the html page overflows over the div with the hero class - do we need to define the height here as `50vh`?